index.js

```diff
@@ -35,16 +41,30 @@
 
 // ***** Question 4 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
+// console.log("%cQuestion 4", "color: red")
+
+// function sameSameButDifferent(num, maybeNum){
+//   if (num === maybeNum)
+    
+//     return "same same"
+  
+//   else if (num == maybeNum)
+    
+//     return "same same (but different)"
+  
+//   else 
+//     return "different"
+  
+// }
```

Comment:
Good. Don't forget your curlies!

```suggestion
function sameSameButDifferent(num, maybeNum){
   if (num === maybeNum) {    
        return "same same"
   }
   else if (num == maybeNum) {
        return "same same (but different)"
   }
   else {
        return "different"
   }
 }
```

Reply:
Another solution for the `else if` would be to convert the string to a number with parseInt to avoid == :

`} else if (num === parseInt(maybeNum)) {`